<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>430</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Задание 17</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/source/fon1.png);</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>347</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>40</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton {
font-size: 14px;
color: rgb(255, 255, 158);
border: 1px solid black;
border-radius: 4px;
background: white;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(238, 238, 255, 255), stop:1 rgba(0, 0, 0, 255));
}

#pushButton:pressed {
font-size: 14px;
color: rgb(255, 255, 158);
background: white;
border: 1px solid black;
border-radius: 4px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(212, 209, 180, 0), stop:1 rgba(0, 0, 0, 255));
}</string>
    </property>
    <property name="text">
     <string>Посчитать</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>312</y>
      <width>60</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>28</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#toolButton {
font-size: 14px;
color: rgb(255, 255, 158);
border: 1px solid black;
border-radius: 4px;
background: white;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(238, 238, 255, 255), stop:1 rgba(0, 0, 0, 255));
}

#toolButton:pressed {
font-size: 14px;
color: rgb(255, 255, 158);
background: white;
border: 1px solid black;
border-radius: 4px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(212, 209, 180, 0), stop:1 rgba(0, 0, 0, 255));
}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>312</y>
      <width>320</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>28</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
color: rgb(255, 255, 158);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Выберите файл с исходным кодом на С:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>290</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>411</width>
      <height>290</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>411</width>
      <height>290</height>
     </size>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
background: transparent;
color: rgb(255, 255, 158);
font-size: 16px;
border: 2px solid rgb(255, 255, 158);
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::FixedColumnWidth</enum>
    </property>
    <property name="lineWrapColumnOrWidth">
     <number>55</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>33</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
background: rgb(109, 110, 117);
font-size: 14px;
}

QMenuBar::item {
font-size: 14px;
color: rgb(255, 255, 158);
padding: 4px 12px;
border: 1px solid black;
border-radius: 4px;
background: white;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(238, 238, 255, 255), stop:1 rgba(0, 0, 0, 255));
 }


QMenuBar::item:pressed {
font-size: 14px;
color: rgb(255, 255, 158);
padding: 3px 6px;
border: 1px solid black;
border-radius: 4px;
background: white;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(212, 209, 180, 0), stop:1 rgba(0, 0, 0, 255));
}</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="styleSheet">
     <string notr="true">background-image: url(:/source/fon.png);
selection-color: rgb(255, 255, 158);
selection-background-color: rgb(109, 110, 117);
color: black;</string>
    </property>
    <property name="locale">
     <locale language="Russian" country="Russia"/>
    </property>
    <property name="title">
     <string>Меню</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Язык</string>
     </property>
     <addaction name="act_ru"/>
     <addaction name="act_en"/>
    </widget>
    <addaction name="act_prog"/>
    <addaction name="act_aut"/>
    <addaction name="menu_2"/>
    <addaction name="separator"/>
    <addaction name="act_esc"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="act_prog">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="act_aut">
   <property name="text">
    <string>Об авторе</string>
   </property>
  </action>
  <action name="act_ru">
   <property name="text">
    <string notr="true">Русский</string>
   </property>
   <property name="iconText">
    <string notr="true">Русский</string>
   </property>
   <property name="toolTip">
    <string notr="true">Русский</string>
   </property>
  </action>
  <action name="act_en">
   <property name="text">
    <string notr="true">English</string>
   </property>
   <property name="iconText">
    <string notr="true">English</string>
   </property>
   <property name="toolTip">
    <string notr="true">English</string>
   </property>
  </action>
  <action name="act_esc">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
